### DISCUSSION TITLE
How to update enum strings?

### DISCUSSION THREAD

**Discussion by guidev** [FlineDev/RemafoX#77] [Help / FAQ] [answered]
What's the best way to remove an enum that's not used anymore?

**Comment by Jeehut** [comment, marked as answer]
@guidev Thank you for the great question!

As of now, the fastest way to remove a specific translation from all languages and also from the generated `enum` file is to remove the translation key (and value) manually from your source languages `.strings(dict)` file. So, for example, if English is your source language and you don't use pluralizations, you can open the `Localizable.strings` file, search for the key name there and simply delete it from the file manually. Then, as soon as you build your project again (assuming you've set up the build script), the RemafoX build script will:
1. Remove the same key from all other languages (given that the "Harmonize with source" option under "Normalizer" is turned on)
2. Re-generate the enum file and effectively remove the related entry from there

If you were looking for a UI to remove a specific key, you might want to upvote this feature: https://github.com/FlineDev/RemafoX/issues/12.
If you were looking for a way to automatically find and remove all unused keys, you can request such a feature [here](https://github.com/FlineDev/RemafoX/issues/new?labels=Feature+Request&template=feature_request.md&title=%5BFeat%5D+%3Cyour+feature+title%3E).

I hope this helps. If you have any further questions, feel free to ask anytime.

**Reply by guidev** [reply]
awesome, thank you!

**Reply by Jeehut** [reply]
Glad that I could help!
